crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -34,19 +34,40 @@ pub struct Visibility {
 
 impl Default for Visibility {
     fn default() -> Self {
+        Self::visible()
+    }
+}
+
+impl Visibility {
+    /// Creates a new [`Visibility`], set as visible
+    pub const fn visible() -> Self {
```

Comment:
Associated constant please.

Reply:
and from https://github.com/bevyengine/bevy/blob/f531a94370755155accd625c58a06f07d33edcce/crates/bevy_transform/src/components/transform.rs#L52